boards/arm/adafruit_feather_nrf52840/doc/index.rst

```diff
@@ -103,33 +132,72 @@ Push buttons
 Programming and Debugging
 *************************
 
-Applications for the ``adafruit_feather_nrf52840`` board configuration
-can be built and flashed in the usual way (see :ref:`build_an_application`
-and :ref:`application_run` for more details).
+The Feather nRF52840 ships with the `Adafruit nRF52 Bootloader`_ which
+is compatible with BOSSA and supports flashing using `UF2`_. This allows
+easy flashing of new images, but does not support debugging the device.
+
+Additionally, if :code:`CONFIG_USB_CDC_ACM` is enabled then the BOSSA
+bootloader will be entered automatically when you run :code:`west flash`.
 
 Flashing
 ========
 
-Flashing Zephyr onto the ``adafruit_feather_nrf52480`` board requires
-an external programmer. The programmer is attached to the SWD header.
+#. Build the Zephyr kernel and the :zephyr:code-sample:`blinky` sample application.
+
+.. tabs::
+
+   .. group-tab:: Express
+
+      .. zephyr-app-commands::
+         :zephyr-app: samples/basic/blinky
+         :board: adafruit_feather_nrf52840
+         :goals: build
+         :gen-args: -DCONFIG_USB_CDC_ACM=y
```

Comment:
Why bother with this? Just make USB CDC the default console and tty device in the dts, doesn't really make sense to use a UART that someone would need to solder to some other UART interface port to actually use

Reply:
will do

Reply:
I tried to change the default in https://github.com/zephyrproject-rtos/zephyr/pull/63819/commits/4964a437d5bcd21224b78679a5ecee3cf7924a4e but this has introduced an error I don't know how to address:

```
gen_isr_tables.py: error: multiple registrations at table_index 39 for irq 39 (0x27)
Existing handler 0x1eb59, new handler 0x1eb59
Has IRQ_CONNECT or IRQ_DIRECT_CONNECT accidentally been invoked on the same irq multiple times?
```

Reply:
First thing is get rid of the label